app/src/main/java/com/babylon/wallet/android/presentation/settings/accountsecurity/ledgerhardwarewallets/AddLedgerDeviceViewModel.kt

```diff
@@ -157,6 +137,7 @@ sealed interface ShowLinkConnectorPromptState {
     data object None : ShowLinkConnectorPromptState
     data class Show(val source: Source) : ShowLinkConnectorPromptState
 
+    // TODO fix this
```

Comment:
at the moment it doesn't cause any issues, and since it requires more work, I will handle it after the release